cumulus_library/__init__.py

```diff
@@ -1,2 +1,2 @@
 """Package metadata"""
-__version__ = "0.4.0"
+__version__ = "0.5.0"
```

Comment:
You made this sound like a breaking change in slack - that the manifest changed in a way that would require study changes.

I'd think that would warrant a major version bump, but I don't know what your versioning policy is. Maybe it's not semantic versioning.

Reply:
if we had a 1.X release, I would make this a 2.X. Since we're still in 0.X, I think [it's ok](https://semver.org/#spec-item-4)?

Reply:
I mean sure, fine - the semantic versioning authorities give you an out because you have avoided bumping to a real version so far. But... you're not alpha software moving fast and loose - you have real partners using this for real things. The 0.x is increasingly a lie.

Reply:
(and more to the point - the goal of semantic versioning is to be kind to consumers by warning about big changes - why not just do that?)

Reply:
per our long in person discussion we agreed to do this, and it does not imply that someone should be writing a press release.